app/controllers/v1/vocabularies_controller.rb

```diff
@@ -3,25 +3,29 @@ class VocabulariesController < ApplicationController
     # GET /vocabularies
     def index
       vocabularies = Vocabulary.all
-      render json: { vocabularies: vocabularies.map(&:to_api) }.to_json
+      render json: URIService::JSON.vocabularies(vocabularies)
     end
 
     # GET /vocabularies/:string_key
     def show
       if (vocabulary = Vocabulary.find_by(string_key: params[:string_key]))
-        render json: vocabulary.to_api, status: 200
+        render json: URIService::JSON.vocabulary(vocabulary), status: 200
       else
-        render json: { errors: [{ title: 'Not Found' }] }.to_json, status: 404
+        render json: URIService::JSON.errors('Not Found'), status: 404
       end
     end
 
     # POST /vocabularies
     def create
-      vocabulary = Vocabulary.new(create_params)
-      if vocabulary.save
-        render json: vocabulary.to_api, status: 201
+      if Vocabulary.find_by(string_key: create_params[:string_key])
```

Comment:
Like the Term case I mentioned above, I feel like it might be easier to catch an ActiveRecord::RecordNotUnique error here instead of checking to see if the Vocabulary already exists.